src/models/Factor.js

```diff
@@ -325,6 +328,7 @@ function (Okta, Q, factorUtil, Util, Errors, BaseLoginModel) {
                   };
                 }
                 return trans.poll(options).then(function (trans) {
+                  self.options.appState.set('lastAuthResponse', trans.data);
```

Comment:
Why was this added?

Reply:
This is after the last poll -> success. The callback doesn't get  called in this case.

Reply:
Is this needed to hide the number challenge?